src/Dashboard/dashboard.jsx

```diff
@@ -40,41 +41,46 @@ export function Dashboard({
 
   if (isAuthenticated) {
     return (
-      <div className="container Dashboard">
-        <div className="row align-items-center">
-          <div className="col-12 col-md-6 align-self-center">
-            <h2 className="welcomeUser light">Overview</h2>
-          </div>
-          <div className="col-auto">
-            <Link className="uploadBtn btn btn-primary" to="/upload">Upload Data</Link>
-          </div>
-          <div className="col-auto">
-            <Link className="downloadBtn btn btn-primary" to="/download">Download/View Data</Link>
-          </div>
-          {featureAvailable.dashboardEditable ? editBtn : ''}
-        </div>
-        <div className="row">
-          <div className="col">
-            <h3 className="divHeader">
-              {`${userDisplayName}, ${siteName}`}
-            </h3>
-          </div>
-        </div>
+      <div className="container-fluid">
         <div className="row">
-          { disconnectComponents ? <PreviousUploadsTable previousUploads={previousUploads} /> : <PreviousUploadsTableConnected />}
-          <PreviousUploadsGraph previousUploads={previousUploads} />
-        </div>
-        <div className="row">
-          <div className="col">
-            <h3 className="divHeader">
-              All Sites
-            </h3>
+          <Sidebar />
+          <div className="col-md-9 ml-sm-auto col-lg-10 px-4 Dashboard">
+            <div className="row align-items-center">
+              <div className="col-12 col-md-6 align-self-center">
+                <h2 className="welcomeUser light">Overview</h2>
+              </div>
+              <div className="col-auto">
+                <Link className="uploadBtn btn btn-primary" to="/upload">Upload Data</Link>
+              </div>
+              <div className="col-auto">
+                <Link className="downloadBtn btn btn-primary" to="/download">Download/View Data</Link>
+              </div>
+              {featureAvailable.dashboardEditable ? editBtn : ''}
+            </div>
+            <div className="row">
+              <div className="col">
+                <h3 className="divHeader">
+                  {`${userDisplayName}, ${siteName}`}
+                </h3>
+              </div>
+            </div>
+            <div className="row">
+              { disconnectComponents ? <PreviousUploadsTable previousUploads={previousUploads} /> : <PreviousUploadsTableConnected />}
```

Comment:
Note: will need to change colors set in /src/lib/colors.js to match the new theme. That file is where the widget graphs pull colors from. I can change that in the next branch though.

Reply:
That's great! Thanks!